### ISSUE TITLE
[ENTRY] Quantum enhanced convolutional filter

### ISSUE THREAD

**Issue by RicardoGaGu** [XanaduAI/QHack2021#79]
### Team Name: 
CCH
### Project Description: 

The emerging field of hybrid quantum-classical algorithms joins CPUs and QPUs to speed-up/improve specific calculations within a classical algorithm. This allows for shorter quantum executions that are less susceptible to the cumulative effects of noise and that run well on today’s devices. This is why we intend to explore the performance of a hybrid convolutional neural network model that incorporates a trainable quantum layer, effectively replacing a convolutional filter, in both quantum simulators and QPU.

Our team proposes to design a trainable quantum convolutional filter in a quantum-classical hybrid neural network, appealing for the NISQ era, inspired by these papers: Hybrid quantum-classical Convolutional Neural Networks [1] and Quanvolutional Neural Networks [2] , but generalizing these previous works to use cloud based QPU.

Here is a list of the expected outcomes/ questions to address of this project:

    Complete benchmarking of a quantum convolutional filter (Encoding of data + variational ansatz) embedded in a classical neural network, in the context of an image classification task with the MNIST dataset.

    Example of complete workflow for training a quantum-classical CNN interfacing Pennylane with TensorFlow/Pytorch for automatic differentiation of the quantum and classical layers, and amazon braket for running the workflow on a QPU.

    With the current noise level in cloud-based QPU, what size/depth of the parametrized quantum circuits is expressive enough without performance being buried under noisy conditions. Can we achieve a significant advantage (in terms of evaluation metrics for a fixed number of quantum vs classical parameters/weights) with today’s QPU?

    Visual exploration of convolved features ( output of filters) with both quantum and classical convolutional filters.

### Presentation: 
[https://github.com/KetpuntoG/QFilters](https://github.com/KetpuntoG/QFilters)

### Source code: 
[https://github.com/KetpuntoG/QFilters](https://github.com/KetpuntoG/QFilters)

**Comment by co9olguy** [comment]
Thanks for the submission! We hope you have enjoyed participating in QHack :smiley: 

We will be assessing the entries and contacting the winners separately. Winners will be publicly announced sometime in the next month.

We will also be freezing the GitHub repo as we sort through the submitted projects, so you will not be able to update this submission.
